src/wrappers/tasks.ts

```diff
@@ -106,4 +106,31 @@ export default class {
 
     return data;
   }
+
+  public async clone(taskID: string): Promise<Task> {
+    const original = await this.get(taskID);
+
+    const createdTask = await this.create(original.org || "", original.flux);
+
+    if (!createdTask || !createdTask.id) {
+      throw new Error("Could not create task");
+    }
+
+    await this.cloneLabels(original, createdTask);
+
+    return this.get(createdTask.id);
+  }
+
+  private async cloneLabels(originalTask: Task, newTask: Task): Promise<Label[]> {
+    if (!newTask || !newTask.id) {
+      throw new Error("Cannot create labels on invalid task");
+    }
+
+    const labels = originalTask.labels || [];
+    const pendingLabels = labels.map(async (label) => this.addLabel(newTask.id || "", label as Label));
```

Comment:
you should have to type that label as the second argument, if you are seeing an issue from this we need to fix the root cause.